.gitignore

```diff
@@ -258,3 +258,5 @@ loadtest/*.tmp/
 result*
 
 .idea/
+
+.vscode/settings.json
```

Comment:
Could you please revert this change? We have ignored most files under the `.vscode` directory. This specific file could be used for sharing project specific settings across developers working on the project, on a later date.

In fact, if you think such a thing would be helpful for you, I'll include project specific settings that can be followed across by developers working on the project, in a separate PR.

Reply:
of course, I can. If i do come across this often, will remove a new issue.